test/skills/run-headless-skill-test.sh

```
@@ -0,0 +1,125 @@
+    --allowed-tools=all \
+    --add-dir "$WORK_DIR" \
+    --permission-mode bypassPermissions \
+    2>&1 | tee "$OUTPUT_FILE" >&2
+fi
+
+SESSION_DIR_NAME="$(printf '%s' "$ROOT_DIR" | sed 's#/#-#g' | sed 's#^-##')"
+SESSION_DIR="$HOME/.claude/projects/$SESSION_DIR_NAME"
+
+# Find the most recently modified .jsonl file (within last hour).
+# Use -Bmin on macOS (BSD find) or -mmin on GNU find.
+if find --version 2>/dev/null | grep -q 'GNU'; then
```

Shellcheck (0.11.0)

[info] Some finds don't have a default path. Specify '.' explicitly.

(SC2185)